sandworm/lib/web.ml

```diff
@@ -0,0 +1,63 @@
+open Tyxml.Html
```

Comment:
Generally I'd try to avoid opens, better do `module T = Tyxml.Html` which makes it much easier to know where which name is from when reviewing code while not creating significantly more work typing code.

Reply:
Fair enought :smile: